src/main/java/org/cactoos/map/Grouped.java

```diff
@@ -57,16 +57,19 @@ public Grouped(
         final Function<T, K> keys,
         final Function<T, V> values
     ) {
-        super(() -> {
-            final Stream<T> stream = StreamSupport.stream(
-                list.spliterator(), false
-            );
-            return stream.collect(
-                Collectors.groupingBy(
-                    keys,
-                    Collectors.mapping(values, Collectors.toList())
-                )
-            );
-        });
+        super(
+            new MapOf<>(
```

Comment:
@andreoss normally you don't need to go through `MapOf`, the `stream.collect` returns a `Map` that can be passed directly to `super`. Right?

Reply:
@victornoel Done